liquidprompt

```diff
@@ -1258,6 +1261,200 @@ _lp_error_color() {
     lp_error_color="${LP_COLOR_ERR}${lp_error}${NO_COL}"
 }
 
+_lp_error_meaning() {
+    # Reasonable codes for a POSIX env.
+
+    _lp_error || return "$?"
```

Comment:
The issue with checking for the error code in this way is that if `LP_ENABLE_ERROR` is false, this function will be disabled as well. I think it makes sense to allow users to enable error meanings without enabling the printing of the raw code.

I would change this check to this:
```suggestion
    (( LP_ENABLE_ERROR_MEANING )) || return 2
    (( lp_error != 0 )) || return 1
```